ebos/eclwriter.hh

```diff
@@ -376,6 +385,7 @@ public:
             bool enableDoublePrecisionOutput = EWOMS_GET_PARAM(TypeTag, bool, EclOutputDoublePrecision);
             const Opm::data::Solution& cellData = collectToIORank_.isParallel() ? collectToIORank_.globalCellData() : localCellData;
             const Opm::data::Wells& wellData = collectToIORank_.isParallel() ? collectToIORank_.globalWellData() : localWellData;
+            const Opm::data::Group& groupData = collectToIORank_.isParallel() ? collectToIORank_.globalGroupData() : localGroupData;
```

Comment:
I don't think this `groupData` object is actually used, since the relevant quantities have already been calculated in `evalSummaryState()` and put into the `summaryState()` object.  The restart writer can retrieve the values from there.

Reply:
I agree and I have removed this object from the code - thank you for the correction!

Reply:
> I agree and I have removed this object from the code

Thanks a lot!  I'll launch a data-update now.  I expect we'll be able to merge this later today.